### ISSUE TITLE
ERROR: module not found: `aes_x86_64'

### ISSUE THREAD

**Issue by CorruptCommit** [archzfs/archzfs#316]
When attempting to upgrade to zfs-linux 0.8.2_5.4.2.arch1.1-1 I get the following error:

```sh
$ uname --kernel-release
5.3.13-arch1-1

$ zfs -V
zfs-0.8.2-1
zfs-kmod-0.8.2-1

$ yay -Syu
...
:: Running post-transaction hooks...
(1/8) Creating temporary files...
(2/8) Arming ConditionNeedsUpdate...
(3/8) Updating module dependencies...
(4/8) Install DKMS modules
(5/8) Updating linux initcpios...
==> Building image from preset: /etc/mkinitcpio.d/linux.preset: 'default'
  -> -k /boot/vmlinuz-linux -c /etc/mkinitcpio.conf -g /boot/initramfs-linux.img
==> Starting build: 5.4.2-arch1-1
  -> Running build hook: [base]
  -> Running build hook: [udev]
  -> Running build hook: [autodetect]
  -> Running build hook: [modconf]
  -> Running build hook: [block]
  -> Running build hook: [keyboard]
  -> Running build hook: [keymap]
  -> Running build hook: [encrypt]
  -> Running build hook: [resume]
  -> Running build hook: [filesystems]
  -> Running build hook: [fsck]
==> ERROR: module not found: `aes_x86_64'
==> Generating module dependencies
==> Creating gzip-compressed initcpio image: /boot/initramfs-linux.img
==> WARNING: errors were encountered during the build. The image may not be complete.
==> Building image from preset: /etc/mkinitcpio.d/linux.preset: 'fallback'
  -> -k /boot/vmlinuz-linux -c /etc/mkinitcpio.conf -g /boot/initramfs-linux-fallback.img -S autodetect
==> Starting build: 5.4.2-arch1-1
  -> Running build hook: [base]
  -> Running build hook: [udev]
  -> Running build hook: [modconf]
  -> Running build hook: [block]
==> WARNING: Possibly missing firmware for module: wd719x
==> WARNING: Possibly missing firmware for module: aic94xx
  -> Running build hook: [keyboard]
  -> Running build hook: [keymap]
  -> Running build hook: [encrypt]
  -> Running build hook: [resume]
  -> Running build hook: [filesystems]
  -> Running build hook: [fsck]
==> ERROR: module not found: `aes_x86_64'
==> Generating module dependencies
==> Creating gzip-compressed initcpio image: /boot/initramfs-linux-fallback.img
==> WARNING: errors were encountered during the build. The image may not be complete.
error: command failed to execute correctly
(6/8) Updating icon theme caches...
(7/8) Rebuilding certificate stores...
(8/8) Updating the desktop file MIME type cache...
```

**Comment by CorruptCommit** [comment]
I am still getting this error upgrading from 0.8.2_5.3.13.1.1-1 to 0.8.2_5.4.5.arch1.1-1.  Please let me know if any addition is needed.

**Comment by minextu** [comment]
I don't think this is related to archzfs. Maybe you have `aes_x86_64` in your `/etc/mkinitcpio.conf`?

**Comment by CorruptCommit** [comment]
@minextu, thanks for the reply.

> Maybe you have `aes_x86_64` in your `/etc/mkinitcpio.conf`?

Yup

> I don't think this is related to archzfs.

Doing some searching today and stumbled on [this bug][0].

I removed `aes_x86_64` from `/etc/mkinitcpio.conf` and was able update without any errors.  When I rebooted into the new kernel, both networking and zfs were broken.  I don't have the time to troubleshoot today, but I will report back when I have time. 

[0]: https://bugs.archlinux.org/task/64721
